relay/network_graph/interests.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+def calculate_interests(balance, rate, old_time, new_time):
+    return balance * (new_time - old_time) / (60 * 60 * 24 * 365) * rate / 100000
+
+def apply_interests(a, b, data):
+    """Applies the interest to the balance in data. The interpretation of data depends on whether a < b.
+    a and b play symmetrical roles"""
+    # apparently not, need to refactor
+    new_time = time.time()
```

Comment:
I'd like to see the time get passed in as an argument. Also isn't data['m_time'] in days and time.time() is in seconds?

Reply:
Everything should be in seconds, the smart contract uses seconds to calculate interests (at least in the interest branch).